radix-engine/src/system/payload_validation.rs

```diff
@@ -68,8 +68,6 @@ impl<'s, 'a, Y: KernelApi<SystemConfig<V>>, V: SystemCallbackObject> TypeInfoLoo
                 }
             }
             TypeInfoSubstate::KeyValueStore(_) => TypeInfoForValidation::KeyValueStore,
-            TypeInfoSubstate::Index => TypeInfoForValidation::Index,
```

Comment:
Can we now no longer have these as separate "Owned" things?

Reply:
Yup exactly. If needed in the future, could always add it.